### ISSUE TITLE
mouse placement on bottom right on boot and sometimes can't enter password (to enter password cursor need to be pointer at password feild)

### ISSUE THREAD

**Issue by kritan560** [elementary/switchboard-plug-mouse-touchpad#158]
![IMG_20201217_153150](https://user-images.githubusercontent.com/75132366/102471712-a481ed00-407d-11eb-84b0-4eea32997cff.jpg)
![IMG_20201217_153139](https://user-images.githubusercontent.com/75132366/102471728-a77cdd80-407d-11eb-9cb3-34101f22f303.jpg)

**Comment by hanaral** [comment]
@JoseExposito Isn't this something to do with mutter? It's always the same spot on gnome too

**Comment by JoseExposito** [comment]
Yes, this is a Mutter thing. I don't think we can do much about it. I just set the "Out of Scope" label because it is not _our_ bug, but maybe someone could fix it upstream.
